docs/roles.md

```diff
@@ -0,0 +1,44 @@
+# Party Corgi Roles and How to Get Them
+
+In the [Party Corgi Discord](https://discord.gg/partycorgi) we take advantage of Discord's rich support for roles to identify and highlight different community members and leaders. This includes the following list:
+
+* adventure-club: *
+* Emoji Curator
+* Maintainer
+* Moderator
+* Nitro Booster
+* Streamer
+* Tummler
+
+## Adventure Clubs
+
+> Knowledge Adventure Clubs are the intersection of book clubs intersects with Dungeons & Dragons.
```

Comment:
“book clubs and Dungeons” 

Can the grammar be corrected here? I’ll probably update the source too :)

Reply:
oop yeah for sure

Reply:
```suggestion
> Knowledge Adventure Clubs are the intersection of book clubs and Dungeons & Dragons.
```